homeassistant/components/daikin/switch.py

```diff
@@ -0,0 +1,77 @@
+"""Support for Daikin AirBase zones."""
+import logging
+
+from homeassistant.helpers.entity import ToggleEntity
+
+from . import DOMAIN as DAIKIN_DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+ZONE_ICON = 'mdi:home-circle'
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
There's less overhead if we make this a coroutine and call it `async_setup_platform`. The thread pool won't be involved then.